### PR TITLE
/tg/ garbage subsystem port

### PR DESCRIPTION
**Pull request by Tsurupeta**
## About the Pull Request

(actually from vore)

## Changelog

:cl:
code: Ported GC from /tg/
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
me when i have no idea what this does

**Comment by checkraisefold** [comment]
no issues on playtest

File: code/__defines/time.dm [hunk 1]
```diff
@@ -4,6 +4,9 @@
 ///displays the current time into the round, with a lot of extra code just there for ensuring it looks okay after an entire day passes
 #define ROUND_TIME ( "[world.time - SSticker.round_start_time > MIDNIGHT_ROLLOVER ? "[round((world.time - SSticker.round_start_time)/MIDNIGHT_ROLLOVER)]:[worldtime2text()]" : worldtime2text()]" )
 
+#define MILISECOND * 0.01
```
Comment (MACIEKBAKI) [inline code/__defines/time.dm]: ah yes, milisecond, with one l

**Review by MACIEKBAKI — APPROVED** [review]
it looks alright* and i've heard it works alright


*to my untrained eye

**Pull request merged**